docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.rst

```diff
@@ -35,3 +35,190 @@ Iron Deficiency Risk Effects
+
+Chris Murray and Theo Vos suggested that this approach be used for estimating the impact of interventions that act on hemoglobin concentration on maternal disorders.
+
+Notably, this alternative approach was found to estimate similar (but slightly greater) intervention impact to the standard approach using the location-specific TMREL exposure estimation approach specific and exposure parameters specific to the pregnant and lactating population.
+
+.. todo::
+
+  Cite the maternal anemia project repo here.
+
+Dietary Iron Deficiency
++++++++++++++++++++++++
+
+The dietary iron deficiency cause in GBD 2019 has a population attributable fraction of 100% with the iron deficiency risk factor. This risk-outcome relationship is not described in full in this document page.
+
+When choosing to model dietary iron deficiency, consideration should be paid to the other causes of iron deficiency (iron responsive anemias), described on the :ref:`Anemia Impairment Documentation Page <2019_anemia_impairment>`.
+
+Vivarium Modeling Strategy
+--------------------------
+
+Maternal Disorders
+++++++++++++++++++++
+
+The recommended approach for modeling the relationship between hemoglobin and maternal disorders is described in the `Alternative Approach`_ section in this document. Therefore, there is no need to use the PAF in order to apply the relative risks.
+
+Instead, the relative risks should be applied to both the YLD (or incidence) and YLL (CSMR or EMR) rate for maternal disorders (or a specific modeled individual maternal disorder), such that:
+
+.. math:: 
+
+    rate_i = rate_\text{GBD} * 1 / e^{\text{ln(RR)} * (Hgb_i - Hgb_\text{GBD})}
+
+Where, 
+
+.. list-table:: Parameter Definitions
+   :widths: 5 5 5
+   :header-rows: 1
+
+   * - Parameter
+     - Definition
+     - Note
+   * - :math:`Hgb_i`
+     - Hemoglobin value for an individual pregnant simulant in g/dL
+     - Needs unit conversion! (Exposure typically defined in g/L)
+   * - :math:`Hgb_\text{GBD}`
+     - Mean hemoglobin value among the pregnant/lactating population from GBD in g/dL
+     - Needs unit conversion! (Exposure typically defined in g/L)
+   * - :math:`rate_i`
+     - Rate for an individual simulant
+     - Rate can be incidence, EMR, CSMR, etc.
+   * - :math:`rate_\text{GBD}`
+     - Rate from GBD 
+     - Rate can be incidence, EMR, CSMR; consider whether this should be among total population or pregnant population in maternal disorders cause document
+   * - :math:`RR`
+     - Relative risks for iron deficiency and maternal disorders
+     - Should be constant for all age-groups and causes within maternal disorders group, can choose any
+
+.. list-table:: Relative Risks
+   :widths: 5 5 5
+   :header-rows: 1
+
+   * - Exposure
+     - Relative Risk
+     - Note
+   * - Per unit deficit of hemoglobin (g/dL)
+     - 1.252472 (0.920269, 0.701621)
```

Comment:
Is this "value (lower, upper)", or something else?  Because the values in parentheses are both smaller than the initial number...

Reply:
Also, I would like to see the definition of the relative risk written down somewhere so I can interpret what these numbers mean.